scripts/facts-tracker/index.js

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var fs = require('fs');
+var execSync = require('child_process').execSync;
+
+function escape(value) {
+  return '\'' + value.replace(/'/g, "'\\''") + '\'';
+}
+
+function exec(command) {
+  console.log('>', command.replace(process.env.GITHUB_TOKEN, '************'));
+  return execSync(command).toString();
+}
+
+if (process.env.TRAVIS_REPO_SLUG) {
+  if (process.env.TRAVIS_BRANCH !== 'master') {
+    console.error('facts-tracker: Branch is not master, exiting...');
+    process.exit(1);
+  }
+
+  if (process.env.TRAVIS_PULL_REQUEST !== 'false') {
+    console.error('facts-tracker: This is a pull request, exiting...');
+    process.exit(1);
```

Comment:
I have the habit of exiting with a non-zero values in this kind of tests, but you are right, this time we actually want to exit with 0 :)